build.sh

```diff
@@ -111,12 +111,12 @@ SRC_ORG="${SRC_ORG-netbox-community}"
 SRC_REPO="${SRC_REPO-netbox}"
 BRANCH="${1}"
 URL="${URL-https://github.com/${SRC_ORG}/${SRC_REPO}.git}"
+NETBOX_PATH="${NETBOX_PATH-.netbox}"
 
 ###
 # fetching the source
 ###
-if [ "${2}" != "--push-only" ] ; then
-  NETBOX_PATH="${NETBOX_PATH-.netbox}"
+if [ "${2}" != "--push-only" ] && [ -z "$SKIP_GIT" ] ; then
```

Comment:
Just a very minor thing: Except for the `$DRY` prefixes this file uses the syntax with the curly braces, i.e. `${VARNAME}`. Could you change the variable name here?

Reply:
Certainly! Done